pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +53,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
+      --disable-R-framework
```

Comment:
@peti AFAIK that makes R to build similarly to Linux. Without the build defaults to build R as framework on OSX (which failed to compile packages because of R binary was not found) and would probably also need the R.app to be build for an usable app.

I'm just an R user, not developer, so I'm not sure about the details.

https://cran.r-project.org/doc/manuals/r-devel/R-admin.html#Frameworks